controllers/fcmToken.js

```diff
@@ -0,0 +1,22 @@
+const { saveFcmToken } = require("../models/fcmToken");
+
+/**
+ * Route used to get the health status of teh server
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const fcmTokenController = (req, res) => {
+  const { fcmToken } = req.body;
```

Comment:
how will we be sure that user will be passing `fcmToken` in the body ? What if they miss sending `fcmToken` in the request payload.

Reply:
what if someone tries to hit the API with POSTMAN

Reply:
already dsicussed!